### DISCUSSION TITLE
Scaling of applications

### DISCUSSION THREAD

**Discussion by PatricMahler** [qtile/qtile#3444] [Q&A] [answered]
My Problem: 
- All my gtk toolkit and qt toolkit applications are too huge in scale.
- Especially if I put an alacritty terminal (font size 11) next to another gtk/qt app (also font size 11), then the gtk/qt app is like 3 times bigger in scale compared to the terminal (see attached screenshot). Also in reference to the qtile bar, the gtk/qt apps seem just humongous (also in the attached screenshot).

Infos:
- I am running qtile on the archlinux+lightdm in a vmware VM. 
- I am using lxappearance to configure gtk apps and qt5ct to configure qt5 apps.
- I have not configured any dpi or scaling settings. Current screen resolution is 1080p. However, if I use xrandr to set the resolution to 720p and decrease the horizontal space for the app, then the scaling of the application changes, and the terminal font size and the app font size almost get identical.

I would really appreciate any help or ideas to resolve this and make the scale rather static than dynamic.
And if that is not possible at least reduce the scale for gtk/qt apps to represent the true font size it is set to. So that the scaling and size fits to the qtile bar.

![unknown](https://user-images.githubusercontent.com/28370552/159907899-bc0550c8-fcbc-4406-b0be-ab72c1215074.png)
.

**Comment by SweepingBishops** [comment, marked as answer]
I don't think qtile has got to do anything with the scaling of gtk and qt applications.
Qtile only decides where the window is placed, not what happens inside.
> * I am using lxappearance to configure gtk apps and qt5ct to configure qt5 apps.

Check there if you have any scaling options.

**Reply by PatricMahler** [reply]
I used the tool arandr to set the resolution and it also adjusted the dpi and scaling. Now everything is working.
